File: articles/metrics-objectives-and-goodharts-law/fortran/goodhart_risk.f90
program goodhart_risk
  implicit none
  real :: proxy_gap, pressure, gaming, guardrail_penalty, score
  proxy_gap = 0.38
  pressure = 0.88
  gaming = 0.72
  guardrail_penalty = 1.0
  score = (proxy_gap + pressure + gaming + guardrail_penalty) / 4.0
  print *, "goodhart_risk_score=", score
end program goodhart_risk

File: articles/metrics-objectives-and-goodharts-law/java/GoodhartRisk.java
public class GoodhartRisk {
    public static void main(String[] args) {
        double proxyGap = 0.38;
        double pressure = 0.88;
        double gaming = 0.72;
        double guardrailPenalty = 1.0;
        double score = (proxyGap + pressure + gaming + guardrailPenalty) / 4.0;
        System.out.printf("goodhart_risk_score=%.4f%n", score);
    }
}
